### PR TITLE
update azuer log format

### PR DESCRIPTION
**Pull request by lovebaby979**
```docs
update azuer log format
```

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure.go [hunk 1]
```diff
@@ -401,7 +402,8 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
 		// Only controller-manager would lazy-initialize from secret, and credentials are required for such case.
 		if fromSecret {
 			err := fmt.Errorf("no credentials provided for Azure cloud provider")
-			klog.Fatalf("%v", err)
+			klog.ErrorS(nil, "", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure.go:405]:
```suggestion
			klog.ErrorS(err, "")
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure.go [hunk 2]
```diff
@@ -446,11 +448,11 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
...
-		klog.V(2).Infof("Azure cloudprovider using try backoff: retries=%d, exponent=%f, duration=%d, jitter=%f",
-			config.CloudProviderBackoffRetries,
-			config.CloudProviderBackoffExponent,
-			config.CloudProviderBackoffDuration,
-			config.CloudProviderBackoffJitter)
+		klog.V(2).InfoS("Azure cloudprovider using try backoff",
+			"configCloudProviderBackoffRetries",config.CloudProviderBackoffRetries,
+			"configCloudProviderBackoffExponent",config.CloudProviderBackoffExponent,
+			"configCloudProviderBackoffDuration",config.CloudProviderBackoffDuration,
+			"configCloudProviderBackoffJitter",config.CloudProviderBackoffJitter)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure.go:455]: please ensure gofmt.
run hack/update-gofmt.sh and check hack/verify-gofmt.sh.

**Comment by pacoxu** [comment]
/close
dup as #101666

**Pull request closed without merging**